addons/website/static/src/snippets/s_rating/options.js

```diff
@@ -0,0 +1,184 @@
+odoo.define('website.s_rating_options', function (require) {
+'use strict';
+
+const ColorpickerDialog = require('web.ColorpickerDialog');
+var weWidgets = require('wysiwyg.widgets');
+var options = require('web_editor.snippets.options');
+
+options.registry.Rating = options.Class.extend({
+    /**
+     * @override
+     */
+    start: function () {
+        this.faClassActiveCustomIcons = '';
+        this.faClassInactiveCustomIcons = '';
+        this.iconType = this.$target.attr('data-icon');
+        this.activeIconColor = this._ensureCssColor(this.$target.attr('data-active-icon-color'));
+        this.inactiveIconColor = this._ensureCssColor(this.$target.attr('data-inactive-icon-color'));
+        if (this.iconType === 'custom') {
+            this.faClassActiveCustomIcons = this.$target.attr('data-active-custom-icon');
+            this.faClassInactiveCustomIcons = this.$target.attr('data-inactive-custom-icon');
```

Comment:
Just a note: when possible, instead of $target.attr('data-xxx-yyy-zzz'), you can use `$target[0].dataset.xxxYyyZzz` now.